README.md

```diff
@@ -2,53 +2,9 @@
 
 [![contains technical debt](https://forthebadge.com/images/badges/contains-technical-debt.svg)](#)[![made with crayons](https://forthebadge.com/images/badges/made-with-crayons.svg)](#)[![made in byond](https://user-images.githubusercontent.com/5211576/29499758-4efff304-85e6-11e7-8267-62919c3688a9.gif)](#)
```

Comment:
badge is incorrect, you removed all the tech debt

Reply:
well shit you're right